apps/researcher/src/components/add-object-list-form/index.tsx

```diff
@@ -0,0 +1,151 @@
+'use client';
+
+import {useAuth} from '@clerk/nextjs';
+import {addList} from './actions';
+import {useForm, SubmitHandler} from 'react-hook-form';
+import {zodResolver} from '@hookform/resolvers/zod';
+import {insertObjectListSchema} from '@colonial-collections/database/src/db/schema';
+import {useTranslations} from 'next-intl';
+import {useSlideOut, useNotifications} from 'ui';
+import {camelCase} from 'tiny-case';
+
+interface FormProps {
+  communityId: string;
+  userId: string;
+  slideOutId: string;
+}
+
+interface FormValues {
+  name: string;
+  description: string;
+  createdBy: string;
+  communityId: string;
+}
+
+function Form({communityId, userId, slideOutId}: FormProps) {
+  const {
+    register,
+    handleSubmit,
+    setError,
+    formState: {errors, isSubmitting},
+  } = useForm({
+    resolver: zodResolver(insertObjectListSchema),
+    defaultValues: {
+      name: '',
+      description: '',
+      createdBy: userId,
+      communityId,
+    },
+  });
+
+  const t = useTranslations('AddObjectListForm');
+  const {setIsVisible} = useSlideOut();
+  const {addNotification} = useNotifications();
+
+  const onSubmit: SubmitHandler<FormValues> = async listItem => {
+    try {
+      await addList(listItem);
```

Comment:
Or just `list` (to be in sync with apps/researcher/src/components/add-object-list-form/actions.ts)?